src/util/tokenUtils.js

```diff
@@ -0,0 +1,41 @@
+require("dotenv").config();
+
+const validateIdpToken = async (accessToken)=>{
+    try{
+        const response = await fetch(`https://www.googleapis.com/oauth2/v1/tokeninfo?access_token=${accessToken}`);
```

Comment:
For this we could use the methods given by `openid-client` library as well. Then we won't have to have these URLs in the code. Only the discovery document is needed.